config/locales/de.helpers.yml

```diff
@@ -76,19 +76,31 @@ de:
     have_fun: "Viel Erfolg beim Training!"
     closing: "%{appname} Team"
   match_mailer:
-    next_game: "dein Spiel (%{team_home} vs. %{team_away}) im Event %{name}"
+    start_time: "Wann? Am %{start_date} um %{start_time} Uhr"
+    place: "Wo? %{place}"
+    here: "Hier"
+    apply_changes_html: "%{match_link} kannst du Änderungen vornehmen."
+    add_results_html: "%{match_link} kannst du die Ergebnisse eintragen."
     match_notification:
-      info: "startet morgen um %{start_time}. Ihr trefft euch am folgenden Ort: %{place}"
-      subject: "Dein Spiel startet morgen"
+      subject: "Dein Spiel gegen %{team_name} im Event %{event_name} startet morgen."
+      info_html: "dein Spiel gegen %{team_link} im Event %{event_link} startet morgen."
+      info_txt: "dein Spiel gegen %{team_name} im Event %{event_name} startet morgen."
+      have_fun: "Viel Spaß!"
     match_canceled:
-      info: "wurde abgesagt."
-      subject: "Dein Spiel wurde abgesagt"
+      subject: "Dein Spiel gegen %{team_name} im Event %{event_name} wurde abgesagt."
+      info_html: "dein Spiel gegen %{team_link} im Event %{event_link} wurde abgesagt."
+      info_txt: "dein Spiel gegen %{team_name} im Event %{event_name} wurde abgesagt."
+      have_fun: "Viel Erfolg!"
     match_scheduled:
-      info: "wurde geplant."
-      subject: "Dein Match wurde geplant"
+      subject: "Dein Spiel gegen %{team_name} im Event %{event_name} wurde geplant."
+      info_html: "dein Spiel gegen %{team_link} im Event %{event_link} wurde geplant."
+      info_txt: "dein Spiel gegen %{team_name} im Event %{event_name} wurde geplant."
+      have_fun: "Viel Erfolg beim Spielen!"
     match_date_changed:
-      info: "wurde auf %{start_time} verschoben."
-      subject: "Dein Match wurde verschoben"
+      subject: "Dein Spiel gegen %{team_name} im Event %{event_name} wurde verschoben."
+      info_html: "dein Spiel gegen %{team_link} im Event %{event_link} wurde verschoben."
+      info_txt: "dein Spiel gegen %{team_name} im Event %{event_name} wurde verschoben."
+      have_fun: "Viel Erfolg beim Spielen!"
```

Comment:
intuitively "Spielen" as an a German adjetive in noun form would rather be understood as referring to playing with toys, i think. if that makes any sense here, "Sport" or "Spiel" would sound more idiomatic; but that's minor...